easybuild/easyconfigs/r/R/R-3.5.1-foss-2018b-Python-2.7.15.eb

```diff
@@ -559,7 +559,7 @@ exts_list = [
         'checksums': ['9b731397b7166dd54941fb0d2eac6df60c7a483b2e790f7eb15b4d7b79c9d69c'],
     }),
     ('boot', '1.3-20', {
-        'checksums': ['f339ddb7138ad05fe3e19852caf7d369f9c1ab827a70045ff66223e032cd0e06'],
+        'checksums': ['adcb90b72409705e3f9c69ea6c15673dcb649b464fed06723fe0930beac5212a'],
```

Comment:
@smoors This reverts the changes done in #8054?

Reply:
oops..

Reply:
`curl https://cran.r-project.org/src/contrib/Archive/boot/boot_1.3-20.tar.gz 2>/dev/null | sha256sum`
adcb90b72409705e3f9c69ea6c15673dcb649b464fed06723fe0930beac5212a

Reply:
Nevermind, it's indeed correct as is, see #8271